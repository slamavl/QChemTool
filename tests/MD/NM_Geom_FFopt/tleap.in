source leaprc.gaff 
loadAmberPrep init_structure.prepc 
loadamberparams nab_input.frcmod 
saveoff FGR nab_input.lib 
saveamberparm FGR nab_input.prmtop nab_input.inpcrd 
savepdb FGR nab_input.pdb 
quit 
