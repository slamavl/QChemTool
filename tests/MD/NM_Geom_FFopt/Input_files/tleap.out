-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/prep to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/lib to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/parm to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/cmd to search path.
-f: Source tleap.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: ./tleap.in
----- Source: /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff
----- Source of /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/slamav/Software/AMBER/amber14/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.7, Nov 2013)
Loading Prep file: ./init_structure.prepc
Loading parameters: ./nab_input.frcmod
Reading force field modification type file (frcmod)
Reading title:
remark goes here
 Creating nab_input.lib
Building topology.
Building atom parameters.
Checking Unit.
Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
old PREP-specified impropers:
 total 0 improper torsions applied
 0 improper torsions in old prep form
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
 (no restraints)
Writing pdb file: nab_input.pdb
	Quit
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/prep to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/lib to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/parm to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/cmd to search path.
-f: Source tleap.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: ./tleap.in
----- Source: /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff
----- Source of /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/slamav/Software/AMBER/amber14/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.7, Nov 2013)
Loading Prep file: ./nab_input.prepc
Loading parameters: ./nab_input.frcmod
Reading force field modification type file (frcmod)
Reading title:
remark goes here
Building topology.
Building atom parameters.
Checking Unit.
Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
old PREP-specified impropers:
 total 0 improper torsions applied
 0 improper torsions in old prep form
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
 (no restraints)
Writing pdb file: nab_input.pdb
	Quit
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/prep to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/lib to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/parm to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/cmd to search path.
-f: Source tleap.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: ./tleap.in
----- Source: /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff
----- Source of /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/slamav/Software/AMBER/amber14/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.7, Nov 2013)
Loading Prep file: ./nab_input.prepc
Loading parameters: ./nab_input.frcmod
Reading force field modification type file (frcmod)
Reading title:
remark goes here
Building topology.
Building atom parameters.
Checking Unit.
Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
old PREP-specified impropers:
 total 0 improper torsions applied
 0 improper torsions in old prep form
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
 (no restraints)
Writing pdb file: nab_input.pdb
	Quit
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/prep to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/lib to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/parm to search path.
-I: Adding /home/slamav/Software/AMBER/amber14/dat/leap/cmd to search path.
-f: Source tleap.in.

Welcome to LEaP!
(no leaprc in search path)
Sourcing: ./tleap.in
----- Source: /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff
----- Source of /home/slamav/Software/AMBER/amber14/dat/leap/cmd/leaprc.gaff done
Log file: ./leap.log
Loading parameters: /home/slamav/Software/AMBER/amber14/dat/leap/parm/gaff.dat
Reading title:
AMBER General Force Field for organic molecules (Version 1.7, Nov 2013)
Loading Prep file: ./nab_input.prepc
Loading parameters: ./nab_input.frcmod
Reading force field modification type file (frcmod)
Reading title:
remark goes here
Building topology.
Building atom parameters.
Checking Unit.
Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
old PREP-specified impropers:
 total 0 improper torsions applied
 0 improper torsions in old prep form
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
 (no restraints)
Writing pdb file: nab_input.pdb
	Quit
